content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -132,6 +132,31 @@ enforced/disallowed:
 				<br><b>Allowed Values:</b> undefined/nil<br>
 			</td>
 		</tr>
+		<tr>
+			<td>/proc Mount Type</td>
+			<td>
+				The default /proc masks are set up to reduce attack surface, and should be required.<br>
+				<br><b>Restricted Fields:</b><br>
+				spec.containers[*].securityContext.procMount<br>
+				spec.initContainers[*].securityContext.procMount<br>
+				<br><b>Allowed Values:</b> undefined/nil, 'Default'<br>
+			</td>
+		</tr>
+		<tr>
+			<td>Sysctls</td>
+			<td>
+				Sysctls can disable security mechanisms or affect all containers on a host, and should be disallowed except for a whitelisted "safe" subset.
+                A sysctl is considered safe if it is namespaced in the container or the pod, and it is isolated from other pods or processes on the same node.<br>
+				<br><b>Restricted Fields:</b><br>
+				spec.securityContext.sysctls<br>
+				<br><b>Allowed Values:</b><br>
+                kernel.shm_rmid_forced<br>
+                net.ipv4.ip_local_port_range<br>
```

Comment:
nit: Are spaces needed to format the "Allowed Values"? There is some drift right in a number of areas.
nit: The docs capitalize Pod.
nit: In one sentence, 'runtime/default' is set off with back ticks instead of single quotes.

```
On supported hosts, the `runtime/default` AppArmor profile is applied by default. The default policy should prevent overriding or disabling the policy, or restrict overrides to a whitelisted set of profiles.<br>
```

/lgtm

Reply:
https://github.com/kubernetes/website/pull/21492